staging/src/k8s.io/apiserver/pkg/cel/library/cost.go

```diff
@@ -202,7 +203,7 @@ func (l *CostEstimator) CallCost(function, overloadId string, args []ref.Val, re
 
 			return &cost
 		}
-	case "quantity", "isQuantity":
+	case "quantity", "isQuantity", "semver", "isSemver":
```

Comment:
Will it cause cost changes in semver used in earlier releases by DRA?

Reply:
Yes, cost was missing so this impacts DRA.  The expected cost change is small but there is one.  The change is to "semver" and "isSemver" only. For input strings < 10 chars the cost will remain 1.  If the input string is more than 10 chars, these will have a cost > 1.

cc @pohly

Reply:
From a practical perspective, I am not worried about small cost changes even if it affects a beta-level API. Adoption is just starting, so the real-world impact is small.